resource/labeling-app/app.js

```diff
@@ -30,6 +32,7 @@ export default new Vue({
         item: undefined,
         stats: undefined,
         type: undefined,
+        isDynamic: undefined,
```

Comment:
Good job making it more generic. Dynamic means that you can have multiple labels per sample, right? I'd maybe rename it to make it more clear: maybe `isMultiLabel`

Reply:
Fixed in e59a61b